WORKSPACE

```diff
@@ -47,89 +11,25 @@ arm_compiler_configure(
-# `py_toolchain` configuration removed:
-# https://github.com/tensorflow/tensorflow/blob/master/WORKSPACE
-
-http_archive(
-    name = "io_bazel_rules_closure",
-    sha256 = "5b00383d08dd71f28503736db0500b6fb4dda47489ff5fc6bed42557c07c6ba9",
-    strip_prefix = "rules_closure-308b05b2419edb5c8ee0471b67a40403df940149",
-    urls = [
-        "https://storage.googleapis.com/mirror.tensorflow.org/github.com/bazelbuild/rules_closure/archive/308b05b2419edb5c8ee0471b67a40403df940149.tar.gz",
-        "https://github.com/bazelbuild/rules_closure/archive/308b05b2419edb5c8ee0471b67a40403df940149.tar.gz",  # 2019-06-13
-    ],
-)
-
-# Load tf_repositories() before loading dependencies for other repository so
-# that dependencies like com_google_protobuf won't be overridden.
-load("@org_tensorflow//tensorflow:workspace.bzl", "tf_bind", "tf_repositories")
-
-# Please add all new TensorFlow dependencies in workspace.bzl.
-tf_repositories()
+load("@org_tensorflow//tensorflow:workspace3.bzl", "tf_workspace3")
 
-load("@io_bazel_rules_closure//closure:defs.bzl", "closure_repositories")
+tf_workspace3()
 
-closure_repositories()
-
-# Use `swift_rules_dependencies` to fetch the toolchains. With the
-# `git_repository` rules above, the following call will skip redefining them.
-load("@build_bazel_rules_swift//swift:repositories.bzl", "swift_rules_dependencies")
-
-swift_rules_dependencies()
-
-# We must check the bazel version before trying to parse any other BUILD
-# files, in case the parsing of those build files depends on the bazel
-# version we require here.
-load("@org_tensorflow//tensorflow:version_check.bzl", "check_bazel_version_at_least")
-
-# Keep in sync with `.bazelversion`.
-check_bazel_version_at_least("3.0.0")
-
-load("@org_tensorflow//third_party/android:android_configure.bzl", "android_configure")
-
-android_configure(name = "local_config_android")
-
-load("@local_config_android//:android.bzl", "android_workspace")
-
-android_workspace()
-
-# If a target is bound twice, the later one wins, so we have to do tf bindings
-# at the end of the WORKSPACE file.
-tf_bind()
-
-# Required for dependency @com_github_grpc_grpc
-
-load("@com_github_grpc_grpc//bazel:grpc_deps.bzl", "grpc_deps")
-
-grpc_deps()
-
-load(
-    "@build_bazel_rules_apple//apple:repositories.bzl",
-    "apple_rules_dependencies",
-)
-
-apple_rules_dependencies()
-
-load(
-    "@build_bazel_apple_support//lib:repositories.bzl",
-    "apple_support_dependencies",
-)
+load("@org_tensorflow//tensorflow:workspace2.bzl", "tf_workspace2")
 
-apple_support_dependencies()
+tf_workspace2()
 
-load("@upb//bazel:repository_defs.bzl", "bazel_version_repository")
+load("@org_tensorflow//tensorflow:workspace1.bzl", "tf_workspace1")
 
-bazel_version_repository(name = "bazel_version")
+tf_workspace1()
 
-load("@org_tensorflow//third_party/googleapis:repository_rules.bzl", "config_googleapis")
+load("@org_tensorflow//tensorflow:workspace0.bzl", "tf_workspace0")
 
-config_googleapis()
+tf_workspace0()
```

Comment:
The workspace file is *much* simplified, which is great. TF made some changes upstream to allow downstream projects to easily import the entire TF workspace config, which is what we're doing here. This means we no longer need to register our own versions of dependency repos, for example. See e.g. [IREE](https://github.com/google/iree/blob/29610cebc4174225b5d6785703c63e6053409b68/integrations/tensorflow/WORKSPACE) for another example of this.